recipe/meta.yaml

```diff
@@ -7,10 +7,11 @@ package:
 source:
   fn: hdfs3-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/h/hdfs3/hdfs3-{{ version }}.tar.gz
-  sha256: 4e74d3dccfab68596c11d1b2e62420bcc36fbd2692aa2771abc27e08d43bc631
+  sha256: 7d5a4466e7a164431aff027533755e2a48d1b1cf6a0299eb41bd5b51cf60c0b4
 
 build:
-  number: 1
+  number: 0
+  noarch: python
```

Comment:
What is this for? We only actually build for linux, since libhdfs3 isn't fixed for osx yet, and there are no plans at all for windows.

Reply:
This means it works for any python, not any OS.